ansible/roles/ocp4-workload-rhtr-iot-managed/tasks/post_workload.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Implement your Post Workload deployment tasks here
+- name: output workshop info
+  debug:
```

Comment:
Please switch this to use the `agnosticd_user_info` module. You'll need this for bookbag integration anyhow.